sadu-datasource/src/main/java/de/chojo/sadu/datasource/DriverClassNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package de.chojo.sadu.datasource;
```

Comment:
Please add the license header.

You can to it by executing the `licenseFormat` task via gradle.

Reply:
added :)